agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3702,44 +3702,45 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             return false;
         }
 
+        // TODO Move APRadioBasicCapabilities creation to a separate function
```

Comment:
> Can we do that as part of this PR? It can even be a separate commit which moves all the big logic out of the case.

I felt that this PR was already big enough as it is :-) Note that this comment was already there, I just moved it to the appropriate place.

>  At the first step I'll even be happy with a single function `start_autoconfiguration` and move all the logic there.

I'll stick with creating a function just for this tlv, similar to `autoconfig_wsc_add_m1()`